From 23ebcc8e2beb82a7b69a848adb633ad8796b8bde Mon Sep 17 00:00:00 2001
From: Hung Tran <hung.tran.jy@renesas.com>
Date: Sat, 18 Jul 2020 14:31:09 +0700
Subject: [PATCH] Fix glob detection to avoid build error with glibc 2.27 and
 later

Current config detects glob wrongly in glibc 2.27
Thus it tries to build and use internal glob instead, and end up with error
	glob/glob.c:575: undefined reference to `__alloca'

Fix the glob detection base on discussion at
https://lists.gnu.org/archive/html/bug-make/2017-11/msg00001.html

Signed-off-by: Hung Tran <hung.tran.jy@renesas.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 04ec3c7..ca5666b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -351,7 +351,7 @@ AC_CACHE_VAL(make_cv_sys_gnu_glob, [
 #define GLOB_INTERFACE_VERSION 1
 #if !defined _LIBC && defined __GNU_LIBRARY__ && __GNU_LIBRARY__ > 1
 # include <gnu-versions.h>
-# if _GNU_GLOB_INTERFACE_VERSION == GLOB_INTERFACE_VERSION
+# if _GNU_GLOB_INTERFACE_VERSION >= GLOB_INTERFACE_VERSION
    gnu glob
 # endif
 #endif
-- 
1.9.1

